### PR TITLE
Use payloads for mqtt messages

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

File: src/robot_interface/telemetry/payloads.py [hunk 1]
```diff
@@ -88,3 +84,38 @@ class RobotHeartbeatPayload:
...
+    error_description: Optional[str]
+    timestamp: datetime
+
+
+@dataclass
+class InspectionResultPayload:
+    isar_id: str
+    robot_name: str
+    inspection_id: str
+    inspection_path: str | dict
```
Comment (Eddasol) [inline src/robot_interface/telemetry/payloads.py:118]:
```suggestion
    inspection_path: Union[str, dict]
```

File: src/robot_interface/telemetry/payloads.py [hunk 2]
```diff
@@ -82,3 +78,38 @@ class RobotHeartbeatPayload:
...
+
+
+@dataclass
+class InspectionResultPayload:
+    isar_id: str
+    robot_name: str
+    inspection_id: str
+    inspection_path: Union[str, dict]
+    installation_code: str
+    analysis_to_be_run: Optional[str]
```
Comment (sondreo) [inline src/robot_interface/telemetry/payloads.py:114]: Does this have to be optional?

**Review by sondreo — APPROVED** [review]
LGTM! Has been tested locally.

See a minor non-blocking comment.

File: src/robot_interface/telemetry/payloads.py (hunk 2 above)
Comment (Eddasol) [reply src/robot_interface/telemetry/payloads.py:114]: Yes. Its being used the send analysis_type that is defined like: 
`analysis_type: Optional[str] = field(default=None, init=False)`

**Pull request merged**